packages/wallet/frontend/src/components/userCards/UserCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { CopyButton } from '@/ui/CopyButton'
+import { NormalCard } from '../icons/NormalCard'
+import { DetailsCard } from '../icons/DetailsCard'
+import Image from 'next/image'
+import { CardTypes } from '@/lib/context/card'
+
+type UserCardProps = {
+  type: CardTypes
+}
+
+export const UserCard = ({ type }: UserCardProps) => {
+  return (
+    <div className="w-full h-52 items-center flex justify-center">
+      {type === 'normal' ? (
+        <>
+          <NormalCard className="-z-10 absolute" />
+          <span className="text-white uppercase relative top-[73px] -left-[91px]">
+            Timi Swift
```

Comment:
will be replaced with all info of the users card when we have the endpints in place